.github/workflows/pr-copy-ci-hato-bot.yml

```
@@ -23,6 +23,7 @@ jobs:
       - name: Copy CI
         run: |
           worklows_path=.github/workflows
+          rm -f $(find sudden-death/${worklows_path} -type f | grep -v "sudden-death.yml")
```

直さなくてもいいと思うけど find で完結する方が好き

```suggestion
          find sudden-death/${worklows_path} -type f -not -name "*sudden-death.yml" -exec rm -f {} \;
```

---

シンプルで良い感じなので採用